apps/site/src/content/blog/astrolicious-evolved.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: "Astrolicious: Evolved"
+draft: false
+publishDate: 2024-09-18
+authors:
+  - name: Alexander Niebuhr
+    url: https://github.com/alexanderniebuhr
+  - name: Florian Lefebvre
+    url: https://github.com/florian-lefebvre
+---
+
+## We are growing 🌱 🌿 🌳
+
+Astrolicious is growing daily, we have gathered over 100 members on our [Discord](https://chat.astrolicious.dev) server and we are excited to see the enthusiasm inside our community. People are sharing their ideas, asking questions and helping each other with their own blog posts, tutorials or examples.
+
+Additionally we have continued our efforts to get more community members involved with decision making, so we are proud to announce that [Fryuni (Luiz Ferraz)](https://github.com/fryuni), has joined the team to shape our future.
```

Comment:
```suggestion
Additionally, in our continued our efforts to get more community members involved with decision making, we are proud to announce that [Fryuni (Luiz Ferraz)](https://github.com/fryuni), has joined the team to help shape our future.
```
I love Fryuni as much as the next person, but I *assume* you haven't given him free rein here.... 😅